src/main/perl/Configuration.pm

```diff
@@ -213,7 +213,7 @@ sub _create_pid_file
 # given cid drops to zero. it updates %cid_to_number
 #
 
-sub _remove_pid_file ()
+sub _remove_pid_file
 {    #T (indirectly)
```

Comment:
have no clue. i assume some sort of perl critic equalvent (because it is not valid critic comment).